go/vt/srvtopo/srvtopotest/srvtopotest.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package srvtopotest
+
+import (
+	"golang.org/x/net/context"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	vschemapb "vitess.io/vitess/go/vt/proto/vschema"
+	"vitess.io/vitess/go/vt/topo"
+)
+
+// Server is a bare implementation of srvtopo.Server for use in tests
+type Server struct {
```

Comment:
In general the vitess "house style" is (as far as I can tell) to use less generic struct names for test-specific code, i.e. `TestServer` or in this case `PassthroughSrvTopo` or some such.

Reply:
:white_check_mark: sgtm

went with `PassthroughSrvTopoServer` for the 1) non-generic naming + 2) indicating it was a Server impl (direction taken from `ResilientServer` which is the other srvtopo impl